addons/website/controllers/main.py

```diff
@@ -225,6 +225,46 @@ def social(self, social, **kwargs):
             raise werkzeug.exceptions.NotFound()
         return request.redirect(url)
 
+    @http.route('/website/get_suggested_link', type='json', auth="user", website=True)
+    def get_suggested_link(self, needle):
+        current_website = request.website
+        suggested = {}
+        matching_pages_url = []
+
+        if len(needle) > 2:
+            matching_pages = current_website.search_pages(needle, limit=15)
+            for page in matching_pages:
+                page = {
+                    'value': page['loc'],
+                    'label': page['loc'],
```

Comment:
'label': 'name' in loc and '%s (%s)' % (page['name'], page['loc']) or page['loc']